packages/elements-core/src/hoc/withQueryClientProvider.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+import { getDisplayName } from './utils';
+
+const queryClient = new QueryClient({
```

Comment:
Does react-query respect the API's declared cacheability through Cache-Control headers or does it have its own client-side conventions for caching?

Reply:
AFAICT this is one layer above Cache-Control and browser cache. It has it's in memory cache, which we configure. Then when it decides to make a request the browser may still respond from the cache it if has some.